valora-inc/wallet
# feat(statsig): refresh when app becomes active

### Description

Force a statsig refresh when app becomes active by updating the statsig user object with a new custom property, `loadTime`

### Test plan

Manually following the below steps

1. Open app and note variant for any exp (home screen actions control in below example)
2. Change experiment config by forcing a different variant (force home screen actions variant in below example)
3. background and reopen app and check the new variant loads

Also ensured logs don't show an "uninitialized" statsig value.

https://user-images.githubusercontent.com/5062591/236580431-a7474866-1cd9-4a57-8293-1b57ec2e8284.mp4



### Related issues

- Fixes ACT-747

### Backwards compatibility

Yes
